terraform/deploy/app/frontendservice.tf

```diff
@@ -25,11 +25,31 @@ module "ecs-fargate-task-definition" {
   environment = [
     {
       name  = "REACT_APP_UI_LOGIN_URL"
-      value = "https://placeholder-url" # to be amended at a later date (in another ticket)
+      value = "https://${module.ecs-fargate-service.fqdn}"
     },
     {
-      name  = "REACT_APP_API_CONNECT_ENDPOINT"
-      value = "/connect"
+      name  = "REACT_APP_OS_URL"
+      value = "https://${data.terraform_remote_state.orchestration-service.outputs.orchestration_service_fqdn}"
+    },
+    {
+      name  = "REACT_APP_REGION"
+      value = data.aws_region.current.name
+    },
+    {
+      name  = "REACT_APP_USERPOOL_WEBCLIENT_ID"
+      value = module.cognito-app.outputs.app_clients.id
```

Comment:
app_clients implies more than 1? Will this choose the first in the list?

Reply:
Updated